src/main/kotlin/com/sourcegraph/cody/chat/ui/Pluralize.kt

```diff
@@ -1,8 +1,9 @@
 package com.sourcegraph.cody.chat.ui
 
-// Can pluralize "file", "line" and "repo" by adding -s
+// Can pluralize "file", "line", "repo" and "repository"
 fun String.pluralize(count: Int): String =
```

Comment:
I think I like the `ts` version we have of this better that just allows you to either pass the original or an alternative string based on the count. Just because things like "index", "analysis", "matrix", "regex", "appendix" etc all don't work. I've attempted too many times to make good lemmatizers/pluralizers and it's fricking hard.

```ts
//lib/shared/src/common/index.ts
export function pluralize(string: string, count: number | bigint, plural = `${string}s`): string {
    return count === 1 || count === 1n ? string : plural
}
```

Reply:
In my opinion, the proper way to handle grammar inflections is keeping the rules encoded in the string that needs to be inflected. For example:
```
pluralize(1, "repositor[y|ies]") == "1 repository" 
pluralize(2, "repositor[y|ies]") == "2 repositories"
pluralize(3, "blind [mouse|mice]") == "3 blind mice"
pluralize(10, "h4xx3d b0x[|en]") == "10 h4xx3d b0xen"
pluralize(-32767, "sheep[|]") == "-32767 sheep" 
```